screens/map/MapScreen.js

```diff
@@ -26,7 +26,7 @@ import {
 
 const minSnapPoint = 160;
 const midSnapPoint = 325;
-const maxSnapPoint = 460;
+const maxSnapPoint = 488;
```

Comment:
These numbers mean nothing to me, but 👍 
*side not: these are the types of decisions/specifics you want to document! Probably only needs an inline comment here for now

Reply:
Will have to dig through old PRs to find the math but yes will add.